Server/src/BaseQuasarServer.cpp

```diff
@@ -198,6 +204,73 @@ std::string BaseQuasarServer::getApplicationPath() const
     return std::string(serverSettingsPath);
 }
 
+std::string BaseQuasarServer::getProcessEnvironmentVariables() const
+{
+    std::ostringstream result;
+#ifdef __linux__
+    extern char **environ;
```

Comment:
Please add a note explaining that this is standard POSIX interface, and maybe place a link to https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html or any other relevant reference.

Reply:
done